backend/test/unit/VotingEvents.unit.test.js

```diff
@@ -72,3 +73,85 @@ describe('voting-events tests', () => {
         clientSocket.emit('trial-vote', voteForDTO);
     });
 });
+
+describe('night time voting event tests', () => {
+    beforeAll(() => {
+        players = [
+            new Player(null, roomElements.roomID, 'notMafia', RoleEnum.MAFIA, false),
+            new Player(null, roomElements.roomID, 'Doctor', RoleEnum.MEDIC, true),
+            new Player(null, roomElements.roomID, 'Sherlock', RoleEnum.DETECTIVE, false),
+            new Player(null, roomElements.roomID, 'Dude', RoleEnum.CIVILIAN, true),
+        ];
+        MafiaGameMock.addPlayers(players, roomElements.roomID);
+    });
+
+    test('mafia votes w/ no medic vote', (done) => {
+        // Listen for night end to check that player has been killed.
+        clientSocket.on('night-end', (nightEndDTO) => {
+            try {
+                expect(nightEndDTO.playerKilled).toEqual('Dude');
+                done();
+            } catch (error) {
```

Comment:
why does it throw an error?

Reply:
I think when expect fails it throws an error, so the `done` callback isn't executed properly. Before I added the try catch block, jest was not exiting properly because of open handles.